pkg/storage/mock/validator/validator_test.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+package validator_test
+
+import (
+	"testing"
+
+	"github.com/ethersphere/bee/pkg/storage/mock/validator"
+	"github.com/ethersphere/bee/pkg/swarm"
+)
+
+func TestMockValidator(t *testing.T) {
+
```

Comment:
vogon prose always starts with a newline <3

Reply:
I think you and google have it the wrong way around